public/static/locales/en/faq.json

```diff
@@ -224,7 +224,7 @@
       "topic": "Security and Technical Background",
       "question": "What information does Circles collect about me?",
       "answer": [
-        "The only data we store is your email address and username - and we will never share it with anyone. Your wallet’s private key lives offline on your device browser, and is not shared with us.",
+        "The only data we store is your email address and user name - and we will never share it with anyone. Your wallet’s private key lives offline on your device browser, and is not shared with us.",
```

Comment:
My little research suggests `username` to be more correct indeed? https://english.stackexchange.com/a/43440